### PR TITLE
Health check

### PR DESCRIPTION
**Pull request by satish-nubolab**
#21 Add keyword for cluster health check

### CODE REVIEW

File: src/KubeLibrary/KubeLibrary.py [hunk 1]
```diff
@@ -3,6 +3,8 @@
 import ssl
 import urllib3
 from kubernetes import client, config
+import requests
```
Comment (Nilsty) [inline src/KubeLibrary/KubeLibrary.py:6]: requests is not needed, see the comment below.

File: src/KubeLibrary/KubeLibrary.py [hunk 2]
```diff
@@ -11,27 +13,19 @@
 
 class KubeLibrary(object):
     """KubeLibrary is a Robot Framework test library for Kubernetes.
-
```
Comment (Nilsty) [inline src/KubeLibrary/KubeLibrary.py:14]: please leave those empty lines as they improve readability.

File: testcases/healthcheck/healthcheck_kw.robot [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Library           KubeLibrary
+# For incluster execution
+#Library           KubeLibrary    None    True    False
+# For development
+#Library           ../../src/KubeLibrary/KubeLibrary.py  ~/.kube/k3d
+
+*** Keywords ***
+Healthcheck
+    ${output}=  Get Healthcheck
+	Log   \nHealthcheck ${output}:  console=True
```
Comment (Nilsty) [inline testcases/healthcheck/healthcheck_kw.robot:15]: It would be nice if your example keyword could also verify if all checks have passed with ok.

**Review by Nilsty — CHANGES_REQUESTED** [review]

File: src/KubeLibrary/KubeLibrary.py (hunk 1 above)
Comment (satish-nubolab) [reply src/KubeLibrary/KubeLibrary.py]: request module removed.

File: src/KubeLibrary/KubeLibrary.py (hunk 2 above)
Comment (satish-nubolab) [reply src/KubeLibrary/KubeLibrary.py]: Done.

File: testcases/healthcheck/healthcheck_kw.robot (hunk 3 above)
Comment (satish-nubolab) [reply testcases/healthcheck/healthcheck_kw.robot]: Evaluated the response and verified if all checks have passed  with ok.

**Pull request closed without merging**